src/terra-core/TerraCoreDataModel.ttl

```diff
@@ -1,12 +1,11 @@
 # baseURI: http://datamodel.terra.bio/TerraCore
+# imports: http://datamodel.terra.bio/CL_subset.owl
 # imports: http://datamodel.terra.bio/EFO_subset.owl
 # imports: http://datamodel.terra.bio/OBI_assaySubset.owl
 # imports: http://datamodel.terra.bio/OBI_core
 # imports: http://datamodel.terra.bio/TerraCoreValueSets
-# imports: http://datamodel.terra.bio/TerraDCAT_ap
-# imports: http://purl.obolibrary.org/obo/
-# imports: http://www.w3.org/2002/07/owl
```

Comment:
I suspect the rdf:type changes may be due to the missing import http://www.w3.org/2002/07/owl.  I think it's worth adding this import to see if it changes.  And if not, if it will allow you to change it.